### DISCUSSION TITLE
Explain time of challenge : less than 30 seconds or less than 5 seconds ?

### DISCUSSION THREAD

**Discussion by casa10** [Chia-Network/chia-blockchain#3445] [Support]
Hello,
I saw on different forum fews things about time of response when a challenge is asked to you.

Which one can win ? Does the Time must to be less than 30 seconds or less than 5 seconds ? or another ?
Thank you.

In these following Examples (i created from some forum i saw), you have one where time is over 30 seconds and other less : 

2021-04-27T18:24:30.907 harvester chia.harvester.harvester: INFO 8 plots were eligible for farming 65fd441b6b... Found 0 proofs. **Time: 36.41071 s**. Total 3213 plots

2021-04-27T18:26:30.907 harvester chia.harvester.harvester: INFO 8 plots were eligible for farming 65fd441b6b... Found 0 proofs. **Time: 1.51826 s**. Total 800 plots

2021-04-27T18:28:30.907 harvester chia.harvester.harvester: INFO 10 plots were eligible for farming 65fd441b6b... Found 0 proofs. **Time: 7.51826 s**. Total 450 plots

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
